### ISSUE TITLE
Classification task

### ISSUE THREAD

**Issue by grmos** [mk-minchul/AdaFace#61]
I have a closed dataset with 15-20 persons(10 images per person) and i use adaface to extract embendings.I was wondering who is the best classifier? Is SVM (linear Kernel ) a good one ?What do you suggest and how to choose ,knowing that i have cosine simmoilarity as metric?

**Comment by mk-minchul** [comment]
Hi grmos. 
Performing classification with SVM with cosine similarity metric can work. You would have to change the loss function to compute margin to be defined in cosine similarity space. 
Personally, I would simply train a shallow neural network with the extracted embeddings.

**Comment by grmos** [comment]
Thank you  for your response.I ll try your suggestion for training a shallow neural network.In addition i observed that Knn works better than SVM for my dataset.
